src/track/trackrecord.cpp

```diff
@@ -107,24 +107,53 @@ bool copyIfNotEmpty(
 
 } // anonymous namespace
 
+bool TrackRecord::updateSourceSynchronizedAt(
+        const QDateTime& sourceSynchronizedAt) {
+    VERIFY_OR_DEBUG_ASSERT(sourceSynchronizedAt.isValid()) {
+        // Cannot be reset after it has been set at least once.
+        // This is required to prevent unintended and repeated
+        // reimporting of metadata from file tags.
```

Comment:
I don't understand why we cannot change the time after it was set once.

Reply:
Reset time stamp = forget that we have imported the metadata at least once. As a consequence the import would be repeated, thereby losing all intermediate edits in Mixxx. Works as before with the boolean flag, that ensures that metadata is only imported once and then only explicitly upon request.